requirements.txt

```diff
@@ -14,5 +14,5 @@ python3-openid==3.1.0
 requests==2.20.1
 requests-oauthlib==1.3.0 
 selenium==3.141.0 
-tycho-api==1.0.22
+tycho-api==1.2.dev2
```

Comment:
Are there corresponding tycho_api changes? Why the change in version?

Reply:
Good question, it might have crept in from other work, or perhaps that's what it was like when I branched and it reverted.  In either case, the changes are not dependant on tycho.